labelCloud/control/controller.py

```diff
@@ -304,3 +308,10 @@ def key_release_event(self, a0: QtGui.QKeyEvent) -> None:
         if a0.key() == QtCore.Qt.Key_Control:
             self.ctrl_pressed = False
             self.view.status_manager.clear_message(Context.CONTROL_PRESSED)
+
+    def assign_point_label_in_active_box(self) -> None:
```

Comment:
Could be moved to `bbox_controller.py` as this has a reference to `pcd_manager`

Reply:
done here [f19e234](https://github.com/ch-sa/labelCloud/pull/104/commits/f19e234a89d3122a63820a7d66da29ff5ccf394a)